src/wallet/coinselection.cpp

```diff
@@ -268,7 +279,7 @@ bool KnapsackSolver(const CAmount& nTargetValue, std::vector<OutputGroup>& group
 
     if (fFulyMixedOnly) {
         // larger denoms first
-        std::sort(groups.rbegin(), groups.rend(), CompareByPriority());
+        std::sort(groups.rbegin(), groups.rend(), CompareByValue());
```

Comment:
?

Reply:
there is no .Priority() with groups (formerly vCoins). #12257